### PR TITLE
in party members box, make link and underlining consistent  -  Made fixes to Issue  #8016

### PR DESCRIPTION
**Pull request by RussWilkie**
Fixes https://github.com/HabitRPG/habitica/issues/8016
### Changes

---

UUID:

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (crookedneighbor) [inline .project]: Please remove this file.

File: website/views/options/social/group.jade [hunk 2]
```diff
@@ -83,8 +83,8 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
                     span.glyphicon.glyphicon-ban-circle(tooltip=env.t('banTip'))
                 a.media-body
                   span(ng-click='clickMember(member._id, true)')
-                      | {{member.profile.name}}
-                    span(ng-if='group.type === "party"')
+                      | {{member.profile.name}}
```
Comment (crookedneighbor) [inline website/views/options/social/group.jade:86]: Please remove the extra white space you added.

**Review by crookedneighbor — CHANGES_REQUESTED** [review]

File: website/views/options/social/group.jade (hunk 2 above)
Comment (RussWilkie) [reply website/views/options/social/group.jade:86]: Made the changes and re-committed. Would I need to do another pull request?

**Comment by crookedneighbor** [comment]
I manually merged in your change because it looks like you had some trouble rebasing.

This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Comment by RussWilkie** [comment]
@crookedneighbor Sweet! User Id: b8ee1cb8-601e-4928-ad4c-297b815db36b

**Comment by crookedneighbor** [comment]
Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!

**Pull request closed without merging**